### PR TITLE
slowlog command support

### PR DESCRIPTION
**Pull request by winjeg**
need to use in my own projects and it is a common requirement

### CODE REVIEW

File: command.go [hunk 1]
```diff
@@ -640,6 +640,95 @@ func (cmd *FloatCmd) readReply(rd *proto.Reader) error {
 	cmd.val, cmd.err = rd.ReadFloatReply()
 	return cmd.err
 }
+//------------------------------------------------------------------------------
+type SlowLog struct {
+	Id       int64
+	ExecTime int64
+	Time     int64
```
Comment (vmihailenco) [inline command.go]: Can this be more Go friendly

```
Time time.Time
Duration time.Duration
```

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,3 @@
 *.rdb
 testdata/*/
+*.idea/
```
Comment (vmihailenco) [inline .gitignore]: Please revert. These files should be added to global gitignore file.

File: command.go [hunk 3]
```diff
@@ -640,6 +640,95 @@ func (cmd *FloatCmd) readReply(rd *proto.Reader) error {
...
+		}
+		vv := v.(*SlowLog)
+		logs =  append(logs, vv)
+	}
+	return logs, nil
+}
+
+func slowLogParse(rd *proto.Reader, n int64) (interface{}, error) {
+	var slowLog SlowLog
+	id, err := readString2Int(rd)
```
Comment (vmihailenco) [inline command.go]: Is it possible to use rd.ReadInt?

Comment (winjeg) [reply command.go]: At first,  I thought so. but when i use `rd.ReadInt` an error occured
```
redis: can't parse string reply: ":1544754674"
```

so I changed to `rd.ReadString`

File: command.go (hunk 1 above)
Comment (winjeg) [reply command.go]: Hey, can you do me a favor, merge it  and then change it for me?
I deleted the original repository, and can't append the change here now.


I've already prepared the code here:

```go
type SlowLog struct {
	Id       int64
	Time     time.Time
	Duration time.Duration
	Args     []string
}

func slowLogParse(rd *proto.Reader, n int64) (interface{}, error) {
	var slowLog SlowLog
	id, err := readString2Int(rd)
	if err != nil {
		return nil, err
	}
	slowLog.Id = id

	t, err := readString2Int(rd)
	if err != nil {
		return nil, err
	}
	slowLog.Time = time.Unix(int64(t), 0)

	execTime, err := readString2Int(rd)
	if err != nil {
		return nil, err
	}
	slowLog.Duration = time.Millisecond * time.Duration(execTime)

	args, err := rd.ReadArrayReply(stringSliceParser)
	if err != nil {
		return nil, err
	}
	slowLog.Args = args.([]string)
	return &slowLog, nil
}

```

File: command.go (hunk 3 above)
Comment (vmihailenco) [reply command.go]: Then you shoud use ReadIntReply https://github.com/go-redis/redis/blob/master/internal/proto/reader.go#L90

**Pull request closed without merging**